torch/onnx/symbolic_helper.py

```diff
@@ -1364,7 +1364,7 @@ def _set_operator_export_type(operator_export_type):
 
 
 # This function is for debug use only.
-# onnx_shape_inference = False by default.
+# onnx_shape_inference = True by default.
 def _set_onnx_shape_inference(onnx_shape_inference: bool):
```

Comment:
Is this function still used elsewhere? Can we remove it?

Reply:
I think only place left is `_export` api, where `onnx_shape_inference` is one of the argument. So can be altogether removed after its deprecation.